### PR TITLE
feat: Add toggle Wifi feature

### PR DESCRIPTION
**Pull request by akinsolb**
## Change list

Please provide briefly described change list which are you going to propose. 
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] New feature (non-breaking change which adds value to the project)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Documentation
- [x] Have you proposed a file change/ PR with appium to update documentation? 
## Integration tests
- [x] Have you provided integration tests to pass against the beta version of appium? (for Bugfix or New feature)

## Details

Add support for toggling Wifi on Android.

to be squashed and merged after  #423 

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted C# code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: src/Appium.Net/Appium/Interfaces/INetworkActions.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace OpenQA.Selenium.Appium.Interfaces
+{
+    public interface INetworkActions
+    {
+        /// <summary>
+        /// Toggle airplane mode on device
```
Comment (mykola-mokhnach) [inline src/Appium.Net/Appium/Interfaces/INetworkActions.cs:8]: This endpoint only works on Android below version 7

**Comment by mykola-mokhnach** [comment]
I think this PR has conflicts with https://github.com/appium/appium-dotnet-driver/pull/423
Could you please merge the latter and then rebase?

**Comment by akinsolb** [comment]
@mykola-mokhnach it does, I mentioned this in the detail section. Will merge this after #423

**Comment by akinsolb** [comment]
closing this branch & pr. Will re-raise

**Pull request closed without merging**